synapse/lib/stormtypes.py

```diff
@@ -8344,6 +8344,7 @@ async def deref(self, name):
         name = await tostr(name)
         return copy.deepcopy(await self.runt.snap.core.getUserVarValu(self.valu, name))
 
+    @stormfunc(readonly=False)
```

Comment:
This looks like an error? Allowing readonly runtime to set user vars is not ideal.